tests/components/auth/test_init.py

```diff
@@ -579,22 +583,36 @@ def cb():
         assert refresh_token is None
 
 
+@pytest.mark.parametrize(
+    ("delete_token_type"), [None, TOKEN_TYPE_LONG_LIVED_ACCESS_TOKEN, TOKEN_TYPE_NORMAL]
```

Comment:
Make it a dict instead so we can update the json data unconditionally.

Reply:
I'm sorry I don't think I understand.

Reply:
Something like this but formatted correctly.
```suggestion
    ("delete_token_type"), [{}, {"token_type": TOKEN_TYPE_LONG_LIVED_ACCESS_TOKEN}, {"token_type": TOKEN_TYPE_NORMAL}]
```